pycoin/ui/key_from_text.py

```diff
@@ -2,8 +2,8 @@
 
 
 def key_info_from_text(text, networks):
-    metadata = metadata_for_text(text)
     for network in networks:
+        metadata = metadata_for_text(text, checksum_hash=network.ui._base58_checksum_hash)
```

Comment:
Moving this here kind of defeats the purpose. Ugh. Maybe add a new type "as_raw_b58" which doesn't check the checksum.

Reply:
Actually that's not great either.